src/main/java/com/lebediev/movieland/web/controller/dto/MovieDtoForUpdate.java

```diff
@@ -0,0 +1,124 @@
+package com.lebediev.movieland.web.controller.dto;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MovieDtoForUpdate {
```

Comment:
i think that we have the same class MovieDto with the same kind of fields. If you think about int[ ], adding a constructor without parameters will solve this.

Reply:
MovieDto already has "not simple" structure with JsonViews for 3 different scenarios (